app/src/main/res/layout/fragment_gameboard.xml

```diff
@@ -27,6 +27,15 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:listitem="@layout/item_gameboard_header" />
 
+    <Button
+        android:id="@+id/btn_scoreboard"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/scoreboard"
+        android:layout_margin="5dp"
```

Comment:
probably that should use `default_margin_half`. But I haven't checked how it looks, if that's too much we can decrease that.

One general information: It's suggested to always have multipliers of 4 (4, 8, 12, 16, 20, ect) for such stuff.